README.md

```diff
@@ -20,6 +20,15 @@ To compile it  run
 ```
 go build -o ./build/server main.go router.go
 ```
```

Comment:
As main and router live in different directories now the run and build instructions should be adapted accordingly (Why is router run anyways when main alone seems sufficient?)

Reply:
Yes you're right we only need go run main.go etc.